packages/stores/src/mockStore.ts

```diff
@@ -1,7 +1,7 @@
 import merge from 'lodash.merge';
-import { TStores } from '../types';
+import type { TRootStore } from '../types';
```

Comment:
I think TCoreStores is being used.
Have to confirm from `Farzin`

Reply:
Circle CI is breaking because of TRootStore doesnot have all the types required.
Use the TCoreStore.